AlphaWallet/Tokens/Helpers/CachedERC1155ContractDictionary.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  CachedERC1155ContractDictionary.swift
+//  AlphaWallet
+//
+//  Created by Jerome Chan on 12/4/22.
+//
+
+import Foundation
+
+class CachedERC1155ContractDictionary {
+    private var baseDictionary: [AlphaWallet.Address: Bool] = [AlphaWallet.Address: Bool]()
+    private var fileUrl: URL
```

Comment:
These 2 can be `let`s

Reply:
Only fileUrl can be let. The baseDictionary is mutated when an address not in the baseDictionary is added. 
```
39: baseDictionary[address] = result
```